docs/configuration/layers/esri-dynamic.md

```diff
@@ -61,9 +61,11 @@ The URL for the DataBC Layer Catalog.
 ## DynamicLayers Property
 `"dynamicLayers": Array`
 
-A listing of dynamic layer configurations.
-This will typically only contain one dynamic feature, which can be derived from the MPCM Layer Catalog.
-
+A listing of dynamic layer configurations in JSON format. This will typically only contain one dynamic feature. This configuration comes from the MPCM Layer Catalog.
```

Comment:
I would suggest leaving out the 'in JSON format' here and further down.

Reply:
I guess it _is_ pretty obvious. Done.